pynest/examples/Potjans_2014/README.rst

```diff
@@ -1,61 +1,108 @@
-``python example.py``. The output will be saved in the directory ``data``.
+   +--------+--------+--------+
+   | |img1| | |img2| | |img3| |
+   +--------+--------+--------+
+   
+Left: network sketch from [2]_. Middle: raster plot showing spiking activity. Right: firing rates as box plots.
 
-The code can be parallelized using OpenMP and MPI, if NEST has been built with
-these applications `(Parallel computing with NEST) <https://www.nest-simulator.org/parallel_computing/>`__.
-The number of threads (per MPI process) can be chosen by adjusting
-``local_num_threads`` in ``sim_params.py``. The number of MPI processes can be
-set by choosing a reasonable value for ``num_mpi_prc`` and then running the
-script with the following command.
+File Structure
+##############
 
-.. code-block:: python
+* ``run_microcircuit.py``: an example script to try out the microcircuit
+* ``network.py``: the main Network class with functions to build and simulate the network
+* ``helpers.py``: helper functions for network construction, simulation and evaluation
+* ``network_params.py``: network and neuron parameters
+* ``stimulus_params.py``: parameters for optional external stimulation
+* ``sim_params.py``: simulation parameters
+* ``reference_data``: reference data and figures obtained by executing ``run_microcircuit.py`` with default parameters
 
-   mpirun -n num_mpi_prc python example.py
+Running the Simulation
+######################
 
+By default, the variables ``N_scaling`` and ``K_scaling`` in ``network_params.py`` are set to
+`0.1` which is a good choice for running the microcircuit on a local machine.
+``N_scaling`` adjusts the number of neurons and ``K_scaling`` adjusts the indegrees.
+The full network can be run by setting these values to `1`.
+If this is done, the option to print the time progress should be switched off: ``'print_time': False`` in ``sim_params.py``.
 
-The default version of the simulation uses Poissonian input, which is defined
-in the file ``network_params.py`` to excite neuronal populations of the
-microcircuit. If no Poissonian input is provided, DC input is calculated, which
-should approximately compensate the Poissonian input. It is also possible to
-add thalamic stimulation to the microcircuit or drive it with constant DC
-input. This can be defined in the file ``stimulus_params.py``.
+To run the simulation, simply use:
 
-Authors
---------
+.. code-block:: bash
 
-Hendrik Rothe, Hannah Bos, Sacha van Albada
+   python run_microcircuit.py
 
+The output will be saved in the directory ``data``.
 
+The code can be `parallelized <https://nest-simulator.readthedocs.io/en/latest/guides/parallel_computing.html>`_ using OpenMP and MPI, if NEST has been built with these applications.
+The number of threads (per MPI process) can be chosen by adjusting ``local_num_threads`` in ``sim_params.py``.
+The command for running the script with two MPI processes would be:
```

Comment:
`with two MPI processes is:`

Reply:
maybe 'features' instead of 'applications'

Reply:
Done.